README.md

```diff
@@ -2,6 +2,11 @@
 
 # TokenSPICE: EVM Agent-Based Token Simulator 🐟🌪️
 
+<div align="center">
+<img alt="Pytest Unit Testing" src="https://github.com/JohannSuarez/tokenspice/actions/workflows/pytest.yml/badge.svg">
+<img alt="MyPy Static Type Checking" src="https://github.com/JohannSuarez/tokenspice/actions/workflows/mypy.yml/badge.svg">
```

Comment:
It tells me that the static type checking is failing. Is it passing for you? If no, what would be next steps? We don't want checks to be put in that are immediately broken.

Reply:
Hello, Trent! The static type checks have been failing by default even when using MyPy manually outside of this Github workflow. The result of this test is true to the current status of the TokenSPICE repo. I'll have to study more about MyPy and stubs before I can completely tackle the errors.

In the meantime, we can temporarily hide the MyPy badge from the README to hide the error status, or rig it to automatically pass.